components/hero/hero.module.scss

```diff
@@ -5,13 +5,13 @@
   align-items: center;
   display: flex;
   flex-wrap: wrap;
-  margin: 100px auto;
+  margin: 200px auto 10px;
```

Comment:
Please state in rem for consistency

Reply:
there were a lot of places where px was used. @tynandebold was ok for me to convert it to rem. So I have updated in this PR